portfolio/src/main/java/com/google/sps/servlets/contactMeServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-submit")
+public class contactMeServlet extends HttpServlet {
```

Comment:
Class names always start with an uppercase ([style guide](https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names)).

Make sure to rename this file also with an uppercase C, and double-check to see if changing the name caused any other errors.
```suggestion
public class ContactMeServlet extends HttpServlet {
```